priv/repo/migrations/20211201035329_create_likes.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Twitter.Repo.Migrations.CreateLikes do
+  use Ecto.Migration
+
+  def change do
+    create table(:likes) do
+      add :user_id, references(:users, on_delete: :delete_all), null: false
+      add :post_id, references(:posts, on_delete: :delete_all), null: false
+      timestamps()
+    end
+    create index(:likes, [:post_id])
```

Comment:
I think you may want a unique constraint on a user_id and post_id at the database layer to unless you want to allow one user to like the same post many times.

Reply:
Good point! I will do that.

Reply:
In migration: 
`create unique_index(:likes, [:user_id, :post_id], name: :unique_index_by_liker_post)`

In changeset: 
`|> unique_constraint(:one_like_per_post_person, name: :unique_index_by_liker_post)`

Reply:
ok, deleted the names